specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5214 @@
+      "description": "Paginated list of Machine Learning service objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array of Machine Learning compute objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/ServiceResource"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "ACIServiceResponse": {
+      "description": "The response for an ACI service.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ServiceResponseBase"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "containerResourceRequirements": {
+              "description": "The container resource requirements.",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/ContainerResourceRequirements"
+                }
+              ]
+            },
+            "scoringUri": {
+              "description": "The Uri for sending scoring requests.",
+              "type": "string"
+            },
+            "location": {
+              "description": "The location of the service.",
+              "type": "string"
+            },
+            "authEnabled": {
+              "description": "Whether or not authentication is enabled on the service.",
+              "type": "boolean"
+            },
+            "sslEnabled": {
+              "description": "Whether or not SSL is enabled.",
+              "type": "boolean"
+            },
+            "appInsightsEnabled": {
+              "description": "Whether or not Application Insights is enabled.",
+              "type": "boolean"
+            },
+            "dataCollection": {
+              "description": "Details of the data collection options specified.",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/ModelDataCollection"
+                }
+              ]
+            },
+            "sslCertificate": {
+              "description": "The SSL certificate to use if SSL is enabled.",
+              "type": "string"
+            },
+            "sslKey": {
+              "description": "The SSL key for the certificate.",
+              "type": "string"
+            },
+            "cname": {
+              "description": "The CName for the service.",
+              "type": "string"
+            },
+            "publicIp": {
+              "description": "The public IP address for the service.",
+              "type": "string"
+            },
+            "publicFqdn": {
+              "description": "The public Fqdn for the service.",
+              "type": "string"
+            },
+            "swaggerUri": {
```

Comment:
are these user provided or readonly?  If readonly please mark all properties that are readonly appropriately

Reply:
made changes